package.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "snoking-to-benchapp-v2",
     "version": "0.1.0",
+    "homepage": "https://shawncollinge.github.io/snoking-to-benchapp-csv.github.io/",
```

Comment:
Can you take this out for the merge to the main repo? I'm not sure if it will cause anything funky to happen given the URL is different.

Reply:
Removed that and the pre-build stuff I added in packages.json.